<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="comboBoxDatabases">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>10</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать бд</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Выбрать таблицу</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxTables">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>131</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SQL запрос</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="sqlRequest">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>80</y>
      <width>241</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="executeRequestBUTTON">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Выполнить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="getTablesFromChosenDatabaseBUTTON">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>40</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Просмотреть</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="showData">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>761</width>
      <height>251</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="createTableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>211</width>
      <height>111</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="addNewFieldInTableBUTTON">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteChosenFieldFromTableBUTTON">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="createTableBUTTON">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Создать таблицу</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteTableBUTTON">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>260</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить таблицу</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="addValuesWidget">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>140</y>
      <width>311</width>
      <height>111</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Имя</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Значение</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Тип</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="executeAddingValuesBUTTON">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>260</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Вставка значений</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addValueToChosenTableBUTTON">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteValueFromChosenTableBUTTON">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>190</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
